ref/general/hardening-intro.adoc

```diff
@@ -22,49 +22,38 @@ Hardening is a general term that is used to describe the measures that are taken
 
 [#operating-system]
 == Operating system
-Operating system intrusions occur when users with local access attempt to cause damage or extract sensitive information. One example of this type of intrusion is malware being introduced into an operating system, allowing unauthorized access to the operating system and its contents.
+Operating system intrusions occur when users with local access attempt to cause damage or extract sensitive information. One example of this type of intrusion is malware. If malware is introduced into a system, unauthorized users might be able to access the operating system and its contents.
 
-To learn more about operating system vulnerabilities and ways to prevent operating system intrusions, go to <link to operating system topic>.
+To learn more about operating system vulnerabilities and ways to prevent operating system intrusions, go to <link pending>.
 
-=== Best practices
+==== Best practices
 
-===== Keep Open Liberty up to date
-To reduce operating system vulnerabilities, it's important to keep your version of Open Liberty up to date. Open Liberty fix packs are upward compatible, and the zero migration policy allows you to move to a newer fix pack without changing anything in your server configuration. Ensuring that you have a recent version of Open Liberty positions you for link:/docs/ref/general/#security-vulnerabilities.html[critical security fixes].
-
-===== Use user registries that are safe for production
-Basic and Quick Start Security user registries are useful for development environments, but they are not acceptable for readiness testing, QA, or production use. Instead, use an LDAP user registry, federated repositories, or create a custom user registry. If you use an LDAP user registry, maintain your bind password in a separate `include` file, not in the `server.xml` file. If you use a custom registry, use the `UserRegistry` class to implement the registry, and make sure that the registry is thoroughly reviewed before it's used in production.
-
-===== Carefully configure file permissions
-When you configure file permissions, ensure that those who want access to the configuration fall into the appropriate permission bit. People who shouldn't have access at all fall into the `other` permission bit that's marked as `0`.
-
-===== Encrypt passwords
-Use encryption for passwords instead of Base64 encoding. You can use the `securityUtility` command, which provides plain text encryption for Open Liberty.
+* *Keep Open Liberty up to date*: To reduce operating system vulnerabilities, it's important to keep your version of Open Liberty up to date. Open Liberty fix packs are compatible with later versions, and the zero migration policy means that you can move to a newer fix pack without changing anything in your server configuration. Ensuring that you have a recent version of Open Liberty positions you for link:/docs/ref/general/#security-vulnerabilities.html[critical security fixes].
```

Comment:
`Ensuring that you have a recent version of Open Liberty positions you for link:/docs/ref/general/#security-vulnerabilities.html[critical security fixes].`

`Keeping Open Liberty up to date helps ensure that you have the most recent link:/docs/ref/general/#security-vulnerabilities.html[critical security fixes].`

Reply:
should there be a link to DL the most recent version here?

Reply:
Updated.